.eslintrc.json

```diff
@@ -17,20 +17,29 @@
       "error",
       {
         "zones": [
-          // this means that in the src/www/infrastructure folder,
+          // This means that in the src/infrastructure folder,
           // you can't import any code,
-          // with the exception of other files within the src/www/infrastructure folder
+          // with the exception of other files within the src/infrastructure folder.
           {
             "target": "./src/infrastructure",
             "from": ".",
             "except": ["./src/infrastructure", "./node_modules"]
           },
-          // similar to above but for src/www/model, but you can use files from both the
-          // src/www/model and src/www/infrastructure paths
+          // Similar to above but for src/www/model, but you can use files from both the
+          // src/www/model and src/www/infrastructure paths.
           {
             "target": "./src/www/model",
             "from": ".",
-            "except": ["./src/www/model", "./src/www/infrastructure", "./node_modules"]
+            "except": ["./src/www/model", "./src/infrastructure", "./node_modules"]
+          },
+          // Prevents the internals of the outline_server_repository from being used publicly in the app.
+          {
+            "target": "./src/www/app/*.ts",
+            "from": "./src/www/app/outline_server_repository/server.ts"
```

Comment:
So this doesn't work?

```suggestion
            "target": "./src/www/app/*.ts",
            "from": "./src/www/app/outline_server_repository",
            "except": ["./src/www/app/outline_server_repository/index.ts"]
```

Reply:
Yeah :( It results in the following:
![Screen Shot 2022-08-25 at 15 28 35](https://user-images.githubusercontent.com/3759828/186751738-32835369-0c98-4063-93bb-74d758c2519d.png)

Reply:
Maybe ` "except": ["./src/www/app/outline_server_repository"]`?

Reply:
I guess you'd need  `"except": ["./src/www/app/outline_server_repository/**"]`

Reply:
apparently that doesn't work either:
![Screen Shot 2022-08-25 at 16 13 13](https://user-images.githubusercontent.com/3759828/186759799-b23775bc-d077-4959-883d-2e8fab644381.png)